src/Helper/Normalizer.php

```diff
@@ -106,13 +105,21 @@ public static function renderLabel(
     ): string {
         $html = '';
 
+        $tagName = self::iconTagName($iconAttributes);
+
+        unset($iconAttributes['tagName']);
```

Comment:
Was `tagName` taken from the example in the issue only, or is it used anywhere else? How about `tag`?

Reply:
Should it be described somewhere in phpdoc and/or in docs?

Reply:
done.